tuning/trackers/aimstack_tracker.py

```diff
@@ -37,18 +83,22 @@ def get_hf_callback(self):
         exp = c.experiment
         url = c.aim_url
         repo = c.aim_repo
+        run_id_path = c.aim_run_id_export_path
 
         if url is not None:
-            aim_callback = AimCallback(repo=url, experiment=exp)
+            aim_callback = CustomAimCallback(repo=url, experiment=exp)
         if repo:
-            aim_callback = AimCallback(repo=repo, experiment=exp)
+            aim_callback = CustomAimCallback(repo=repo, experiment=exp)
         else:
             self.logger.warning(
                 "Aim tracker requested but repo or server is not specified. "
                 + "Please specify either aim repo or aim server ip and port for using Aim."
             )
             aim_callback = None
 
+        if aim_callback is not None:
```

Comment:
```suggestion
        if aim_callback:
```

Reply:
Its just a preference to keep the check explicit.